neurom/features/morphology.py

```diff
@@ -374,18 +377,25 @@ def trunk_origin_radii(
         return [n.root_node.points[0][COLS.R]
                 for n in iter_neurites(morph, filt=is_type(neurite_type))]
 
-    def _mean_radius(points):
+    def _mean_radius(neurite):
+        points = neurite.root_node.points
         interval_lengths = morphmath.interval_lengths(points)
         path_lengths = np.insert(np.cumsum(interval_lengths), 0, 0)
         valid_pts = np.ones(len(path_lengths), dtype=bool)
         if min_length_filter is not None:
             valid_pts = (valid_pts & (path_lengths >= min_length_filter))
+            if not valid_pts.any():
+                warnings.warn(
+                    "In 'trunk_origin_radii': the 'min_length_filter' value is greater than the "
+                    "path distance of the last point of the last section so the radius of this "
+                    "point is returned."
+                )
+                valid_pts[-1] = True
```

Comment:
Would it be better if you returned early here? If the `min_length_filter` is greater than the path distance, the `max_length_filter` should be as well if specified. 

Therefore, I would:
```python
return points[-1, COLS.R]
```
What do you think?

Reply:
You made me think more about edge cases and indeed there are several weird possibilities. In the last commit I tried to handle them, WDYT?

Reply:
I think it's great now! You still can return early though, can't you?

Reply:
Ah yes I forgot sorry